.github/workflows/ci.yaml

```diff
@@ -66,6 +66,24 @@ jobs:
         run: go mod download
       - name: Test
         run: make test
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Registry Login
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAYIO_USERNAME }}
+          password: ${{ secrets.QUAYIO_PASSWORD }}
```

Comment:
As we are not publishing the docker image, so do we need to do docker login here?

Reply:
I though i needed the step for the image build step but i removed it

Reply:
@chandankumar4 should be all good to go now